python/mxnet/gluon/metric.py

```diff
@@ -24,9 +24,9 @@
 
 import numpy
```

Comment:
Instead of using numpy, we can use `mxnet.numpy` as it runs asynchronously and has GPU support.
The summary states of a metric should be stored on CPU, but if for example data and label are on GPU and input to the metric, we can calculate the sufficient statistics on GPU